server/middleware/validationMiddleware.ts

```diff
@@ -10,9 +10,9 @@ export type FieldValidationError = {
 
 function validationMiddleware(type?: new () => unknown): RequestHandler {
   return async (req, res, next) => {
-    const { additionalConditionId } = req.params
+    const { conditionId } = req.params
     let additionalConditionType
-    if (additionalConditionId) {
+    if (conditionId) {
       additionalConditionType = getAdditionalConditionByCode(req.body.code)?.type as ClassConstructor<unknown>
     }
```

Comment:
Could this be a PSS condition too now?

Reply:
Yes both will go in here as both have a conditionId

Reply:
Just the name - additionalConditionType - suppose it could apply to either, so ok for now.